spec.yaml

```diff
@@ -601,6 +615,28 @@ paths:
         omitted, default configuration values will be applied. For example, if
         no `release_version` is specified, the most recent version is used.
 
+        The number of `availability_zones` affects the total number of nodes
+        that can be created in the cluster. The number of availability zones
+        splits the ip range that can be used for the cluster in multiple smaller
+        ip ranges. The [getInfo](#operation/getInfo) endpoint provides more
+        details about the cluster ip range.
+
+        IP range example:
+
+        If a cluster gets a `/22` (1022 hosts). And the cluster should be
+        spawned across 3 availability zones. The range will then be split up
+        into four `/24` (254 hosts) that can be given to four different
+        availability zones. One range will be stay unused because ranges can
```

Comment:
Maybe reword a bit: `One range will be unused because network addresses must be powers of two. See [CIDR addressing](https://en.wikipedia.org/wiki/Classless_Inter-Domain_Routing).`